sdk/servicebus/service-bus/test/sessionsTests.spec.ts

```diff
@@ -602,3 +602,94 @@ describe("Peek session", function(): void {
     await peekSession(false);
   });
 });
+
+describe.skip("SessionReceiver - disconnects", function(): void {
```

Comment:
.skip()?

Reply:
I'll leave a comment, but yes, this is intentional.